pyproject.toml

```diff
@@ -1,17 +1,107 @@
 [project]
 name = "rockcraft"
+description = "Create ROCKS"
 dynamic = ["version", "readme"]
-
+license = {file = "LICENSE"}
+authors = [
+    {name = "Canonical Ltd.", email = "snapcraft@lists.snapcraft.io"}
+]
+ 
 classifiers = [
     "Development Status :: 2 - Pre-Alpha",
     "Intended Audience :: Developers",
-    "License :: OSI Approved :: GNU Lesser General Public License v3 (LGPLv3)",
+    "License :: OSI Approved :: GNU General Public License v3 (GPLv3)",
     "Operating System :: MacOS :: MacOS X",
     "Operating System :: POSIX :: Linux",
     "Programming Language :: Python :: 3",
     "Programming Language :: Python :: 3.10",
 ]
 requires-python = ">=3.10"
+dependencies = [
```

Comment:
I don't know if I agree with moving the frozen requirements here. The problem is that this ties Rockcraft to transitive dependencies forever, even if we don't need them anymore, because the application needs to freeze all of its requirements.
What was the reasoning to remove the requirements-*.txt files?